napkin-app/src/index.css

```
@@ -1,11 +1,13 @@
+@import "tailwindcss";
+
 :root {
   font-family: system-ui, Avenir, Helvetica, Arial, sans-serif;
   line-height: 1.5;
   font-weight: 400;
 
   color-scheme: light dark;
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)